src/components/VTextField/VTextField.js

```diff
@@ -128,6 +132,23 @@ export default {
   },
 
   methods: {
+    maskedUpdateOnValueChange (val) {
+      const masked = this.maskText(this.unmaskText(val))
+      const value = this.unmaskText(masked)
+
+      this.lazyValue = this.getLazyValue(value)
+      this.$nextTick(() => {
+        this.$refs.input.value = masked
+        this.$emit('input', this.lazyValue)
+      })
+    },
+    updateOnValueChange (val) {
+      this.lazyValue = val
+      this.$emit('input', this.lazyValue)
+    },
+    getLazyValue (val) {
```

Comment:
This name is bad, it doesn't accurately describe what the function is doing. Is this method even needed? You only use it in one place.

Reply:
It's used in 2 places. The other one is at line 92 of `VTextField.js`. Do you prefer not to have this this function? I'll just put the content where it was before.

Reply:
What is it actually doing anyway? It looks like it just casts `val` to a number when it already is one.

Reply:
Actually it casts to the type that the dev enters in `props.value`

Reply:
It was [here](https://github.com/vuetifyjs/vuetify/pull/2099/commits/58524576273d9d9f203d8eb1e06d204cddca1d0c#diff-c2cbb2f11cf8c97e6eff0981f9927269L124), but now it does nothing.

Reply:
OMG what was I thinking when I did that.

You're damn sharp man. I salute